Wordle.Api/Wordle.Api/Models/WordDate.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Wordle.Api.Models;
+
+[Table("WordDates")]
+public class WordDate
+{
+    public int WordDateId { get; set; }
+    public string Date {get; set; }
```

Comment:
```suggestion
    public Datetime Date {get; set; }
```
Should use Datetime instead of string